src/main/java/org/jenkinsci/test/acceptance/po/Slave.java

```diff
@@ -21,7 +22,7 @@ public class Slave extends Node {
     private final String name;
 
     public Slave(Jenkins j, String name) {
-        super(j, j.url("computer/%s/",name));
+        super(j, j.getVersion().isNewerThan(new VersionNumber("2.363")) ? j.url("manage/computer/%s/", name) : j.url("computer/%s/", name));
```

Comment:
can't see a smooth way to do this other than versions so will need this to be adapted if it misses the next weekly.

Most tests passed with the initial PR in 2.360 but one failed as there was some more work done on top of the feature in the core PR for this

Reply:
@timja For posterity, could you be more specific why this change?

Reply:
Tests were asserting the expected url but they were now getting a different one from links changing to include `manage`

correct_redirect_after_save – plugins.OwnershipPluginTest
```
Expected: <http://127.0.0.1:32861/computer/precise_cluster/>
     but: was <http://127.0.0.1:32861/manage/computer/precise_cluster/>
```

latestsBuildsPortlet_correctJobAndBuild – plugins.DashboardViewPluginTest
```
Expected: is <true>
     but: was <false>

at plugins.DashboardViewPluginTest.latestsBuildsPortlet_correctJobAndBuild(DashboardViewPluginTest.java:175)
```

Reply:
Right, but that should be 2.359. Why 2.363? (I was looking at https://github.com/jenkinsci/acceptance-test-harness/pull/869/commits/fa5f58289a133eab18dee981d97f1dfa5df3d39c when I commented)

Reply:
see https://github.com/jenkinsci/acceptance-test-harness/runs/7824521017

it requires something from the open PR. likely the links changing in the table to include manage